compoments/report-repair/emergency-repair.js

```diff
@@ -36,4 +40,66 @@ const EmergencyRepair = () => {
   </div>
 };
 
+const EmergencyRepairFull = ({ goToStep, prevStep }) => {
```

Comment:
I wonder if there should be some MVP-specific Cypress tests as well as full version Cypress Tests.

Reply:
I'll add the full version ones. It makes sense. Thanks! I'm not sure about the MVP, wasn't it said to not make changes to MVP stuff? I might be making this up...

Reply:
Yes, good point. Sorry, I meant to separate the old tests into the an MVP version and add the full version tests as we have done for some other pages.

Reply:
Yep. I don't think that this page has got any tests, so I'll add some, so that we have this page covered.

Reply:
I've added the tests